CHANGELOG.md

```diff
@@ -15,19 +15,63 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
     _Security_ in case of vulnerabilities.
  -->
 
-## [Unreleased](https://github.com/o1-labs/o1js/compare/450943...HEAD)
+## [Unreleased](https://github.com/o1-labs/o1js/compare/f15293a69...HEAD)
+
+## [2.0.0](https://github.com/o1-labs/o1js/compare/f15293a69...HEAD)
+
+### Breaking Changes
```

Comment:
Do we have an upgrade guide to the breaking changes?

Reply:
I don't believe we do, but it is a good idea.

Reply:
see slack